src/components/template/TemplateQuestionEditor.tsx

```diff
@@ -93,7 +111,7 @@ export default function TemplateQuestionEditor(props: {
       ? '&&'
       : '||';
   const dependencyJsx = dependencies.length ? (
-    <>
+    <div>
```

Comment:
Do we need this div element here?

Reply:
Actually, we do, because this div has some styles applied to it.